src/Service.Host/package.json

```diff
@@ -22,7 +22,8 @@
         "setupFiles": [
             "raf/polyfill",
             "./react-animation-test-shim.js",
-            "./setup-enzyme.js"
+            "./setup-enzyme.js",
+            "./jest-setup.js"
```

Comment:
Or perhaps the inclusion of this? Is some default config that understands es6 being overwritten?